reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,294 @@
+|                      | - `placeholder`_                                                                  |
+|                      | - `input`_                                                                        |
+|                      | - `widget`_                                                                       |
++----------------------+-----------------------------------------------------------------------------------+
+| Inherited            | - `data`_                                                                         |
+| options              | - `disabled`_                                                                     |
+|                      | - `inherit_data`_                                                                 |
+|                      | - `invalid_message`_                                                              |
+|                      | - `invalid_message_parameters`_                                                   |
+|                      | - `mapped`_                                                                       |
+|                      | - `read_only`_                                                                    |
++----------------------+-----------------------------------------------------------------------------------+
+| Parent type          | :doc:`form </reference/forms/types/form>`                                         |
++----------------------+-----------------------------------------------------------------------------------+
+| Class                | :class:`Symfony\\Component\\Form\\Extension\\Core\\Type\\DateIntervalType`        |
++----------------------+-----------------------------------------------------------------------------------+
+
+Basic Usage
+-----------
+
+This field type is highly configurable, but easy to use. The most important
+options are ``input`` and ``widget``.
+
+Suppose that you have a ``remindEvery`` field whose underlying interval is a
+``DateInterval`` object. The following configures the ``dateinterval`` type
+for that field as three different choice fields:
+
+.. code-block:: php
+
+    $builder->add('remindEvery', 'dateinterval', array(
+        'input'  => 'dateinterval',
+        'widget' => 'choice',
+    ));
+
+The ``input`` option *must* be changed to match the type of the underlying
+interval data. For example, if the ``remindEvery`` field's data were a
+ISO 8601 duration string you'd need to set ``input`` to ``string``:
+
+.. code-block:: php
+
+    $builder->add('publishedAt', 'date', array(
+        'input'  => 'string',
+        'widget' => 'choice',
+    ));
+
+The field also supports an ``array`` as valid ``input`` option values.
+
+Field Options
+-------------
+
+input
+~~~~~
+
+**type**: ``string`` **default**: ``dateinterval``
+
+The format of the *input* data - i.e. the format that the interval is stored on
+your underlying object. Valid values are:
+
+* ``string`` (e.g. ``P1DT12H``)
```

Comment:
Reading the introduction of this article, it seems that the string must be formatted using `RFC 3339`. If this is correct, I think we should change this phrase by the following:

```
* ``string`` (a string formatted with `RFC 3339`_ standard, e.g. ``P1DT12H``)
```